lib/ui/settings/data/import/analyze_qr_code.dart

```diff
@@ -0,0 +1,292 @@
+                                          vertical: 8,
+                                        ),
+                                        child: Text(
+                                          'OK',
+                                          style: TextStyle(
+                                            letterSpacing: 0.5,
+                                            fontWeight: FontWeight.w500,
+                                            fontSize: 16,
+                                            color: Colors.white,
+                                          ),
+                                        ),
+                                      ),
+                                    ),
+                                  ),
+                                  const SizedBox(
+                                    height: 30,
+                                  ),
+                                ],
+                              ),
+                            ],
+                          );
+                        },
+                      );
+                    }
+                  },
+                ),
+                // Qr code scanner overlay
+                const QRScannerOverlay(),
+
+                // Torch and gallery buttons
+                Positioned(
+                  top: 150,
+                  left: 0,
+                  right: 0,
+                  child: Row(
+                    mainAxisAlignment: MainAxisAlignment.center,
+                    children: [
+                      // Torch button
+                      IconButton(
+                        icon: ValueListenableBuilder(
+                          valueListenable: scannerController.torchState,
+                          builder: (context, state, child) {
+                            switch (state) {
+                              case TorchState.on:
+                                return SvgPicture.asset(
+                                  'assets/custom-icons/icons/flash_on.svg',
+                                );
+                              case TorchState.off:
+                                return SvgPicture.asset(
+                                  'assets/custom-icons/icons/flash_off.svg',
+                                );
+                            }
+                          },
+                        ),
+                        iconSize: 60,
+                        onPressed: () => scannerController.toggleTorch(),
+                      ),
+
+                      // Gallery button
+                      IconButton(
+                        icon: SvgPicture.asset(
+                          'assets/custom-icons/icons/gallery.svg',
+                        ),
+                        iconSize: 60,
+                        onPressed: () async {
+                          final result = await showDialogWidget(
+                            context: context,
+                            title: 'Google Authenticator (saved image)',
```

Comment:
Please extract all strings to the `ARB` files so that they an be translated

Reply:
Done

Reply:
Also, the texts are not confirmed. Once it is confirmed, will add it. Just look into the code and see if texts are correct